openhands/runtime/impl/eventstream/eventstream_runtime.py

```diff
@@ -179,6 +179,7 @@ def __init__(
         self.docker_client: docker.DockerClient = self._init_docker_client()
         self.base_container_image = self.config.sandbox.base_container_image
         self.runtime_container_image = self.config.sandbox.runtime_container_image
+        self.rm_all_containers = self.config.sandbox.rm_all_containers
```

Comment:
no need to add this to `self`--we can just reference `self.config.sandbox.rm_all_containers`

Reply:
Done!